### DISCUSSION TITLE
Blocking unwanted Slingbox traffic - instructions for AT&T Arris BGW210 (router-specific)

### DISCUSSION THREAD

**Discussion by DVoorhees1957** [GerryDazoo/Slinger#268] [General]
Given today's nastiness with Sling/Dish purposefully bricking as many Slingboxes as they can, I thought I'd write down the steps I've taken to protect my Slingbox 350 and Slingbox 500. If you're running the slingbox server software on a different network than your Slingbox(es), you can't just block all WAN traffic to and from your Slingbox. What follows are the steps I took for the router to which the Slingboxes are connected, which is an AT&T Arris BGW210 router. NOTE: Every router is different in terms of how to accomplish port blocking and/or port forwarding; unless you have my specific router, you'll have to "translate" these steps to however your router works.

Because of my configuration, where the slingbox server is on a different network than my Slingboxes, I have to first allow all traffic between my slingbox server device (my laptop) and my Slingboxes. Then, I have to disallow all other traffic to and from my Slingboxes. I had to use "packet filtering to do this, because the Arris BGW210 doesn't seem to have a straightforward way to block communication to or from a specific device. To get to the packet filtering rules on the AT&T Arris BGW210 router, click on "Firewall" in the top menu, then click on "Packet Filter" below that.

1. Under "Packet Filter", I added a "Pass rule" that allows all traffic from my laptop's external IP address to the MAC id of my Slingbox(es). I also added a 2nd "Pass rule" that allows all traffic from the MAC id of my Slingbox(es) to my laptop's external IP address. After they are created, I moved these two rules to the very top of the Packet Filter list so that they get applied first (although this step is frankly optional; these 2 rules simply have to come before the next 2 rules that I will describe). Then be sure to check the box to enable both of these rules.

2. Then, I added a "Drop rule" that drops all other communication from anywhere to the MAC id of my Slingbox(es), and I added yet another "Drop rule" that drops all outbound communication from the MAC id of my Slingbox(es). These 2 rules need to come somewhere **after** the 1st two "Pass rules". Again, be sure to check the box to enable both of these rules.

This combination of 2 "Pass rules" and 2 "Drop rules" seems to achieve the desired objective, which is to allow communication back and forth between my laptop and my Slingbox(es), and to block all other attempted communication to or from my Slingbox(es).

**Comment by flipper2222222** [comment]
for me my ISP supplied router doesn't have too many advanced functions so on 7 November I  deleted the slingbox port forwarding rules, and in the parental controls blocked all WAN traffic for the Slingboxes IP addresses , tested that I could not access via the iphone app. 

both 500 and slingbox pro are still working with slinger, hope this helps someone else.

**Comment by Pthepeg** [comment]
I ended up trying a router from my junk box: Linksys EA6100, cascaded with my ISP router. It was very easy to setup and test, and does what I need. I put the slingbox under ea6100 parental control: always blocks internet access. I put the slingserver in the ea6100 DMZ so its open to all ports. The ISP router has select port forwarding to the ea6100.
